util/GO9PTERM

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+set -e
+
+trap : 2
+
+$HARVEY/util/ufs -root=$HARVEY &
+ufspid=$!
+
+export machineflag=pc
+if [ "$(uname)" = "Linux" ] && [ -e /dev/kvm ]; then
+	export kvmflag='-enable-kvm'
+	export machineflag='pc,accel=kvm'
+	if [ ! -w /dev/kvm ]; then
+		# we don't have access as a regular user
+		export kvmdo=sudo
+	fi
+fi
+
+read -r cmd <<EOF
+$kvmdo qemu-system-x86_64 -s -cpu Opteron_G1 -smp 1 -m 2048 $kvmflag \
+-serial stdio \
+--machine $machineflag \
+-net nic,model=rtl8139 \
+-net user,hostfwd=tcp::5555-:1522 \
+-net dump,file=/tmp/vm0.pcap \
+-redir tcp:9999::9 \
+-redir tcp:17010::17010 \
+-redir tcp:5356::5356 \
+-redir tcp:17013::17013 \
+-append "service=terminal nobootprompt=tcp maxcores=1024 fs=10.0.2.2 auth=10.0.2.2 nvram=/boot/nvram nvrlen=512 nvroff=0 acpiirq=1" \
+-kernel $HARVEY/sys/src/9/amd64/harvey.32bit $*
```

Comment:
Needs "mouseport=ps2 vgasize=1024x768x24 monitor=vesa"
See "getting_started" in wiki. For some reason, the traditional questions made byr termrc won't work.